tests/engine/adhoc_test.go

```diff
@@ -16,7 +15,7 @@ import (
 )
 
 func requireJITSupported(t *testing.T) {
```

Comment:
and now if you want, you can inline the below as it is just as intuitive and a bit more direct

```
if !wazero.JITSupported {
 		t.Skip()
 	}
```

Reply:
already inlined :D